### PR TITLE
build cache for revocable nodes

### PR DESCRIPTION
**Pull request by yahaa**
Build cache for revocable node.

### CODE REVIEW

File: pkg/scheduler/cache/cache.go [hunk 1]
```diff
@@ -684,6 +685,10 @@ func (sc *SchedulerCache) Snapshot() *schedulingapi.ClusterInfo {
 		}
 
 		snapshot.Nodes[value.Name] = value.Clone()
+
+		if value.RevocableZone != "" {
+			snapshot.RevocableNodes[value.Name] = value.Clone()
```
Comment (k82cn) [inline pkg/scheduler/cache/cache.go:690]: `RevolcanoNodes` should be an index, so just one copy of `value.Clone()` should be enough.

Comment (yahaa) [reply pkg/scheduler/cache/cache.go]: Fixed it.

**Pull request merged**